Readme.md

```diff
@@ -1,176 +1,341 @@
-the next module is triggered to start analysing the page.
-
-#### [`src/page-analysis`](src/page-analysis/): page analysis
-
-This extracts and stores information about the page in a given tab, such as:
-- The plain text of the page, mainly for the full-text
-search index.
-- Metadata, such as its author, publication date, etcetera.
-- A screenshot for visual recognition.
-
-#### [`src/overview/`](src/overview/): overview
-
-The `overview` is the user interface that opens in a tab of its own. It is built
-with [React](https://facebook.github.io/react/) and [Redux](http://redux.js.org/),
-which create a somewhat complex but nicely organised application structure.
-
-See [`src/overview/Readme.md`](src/overview/Readme.md) for more details.
-
-#### [`src/search`](src/search/): document search
-
-Functions for finding relevant knowledge in the user's memory. Currently
-provides a simple full-text keyword search through visited pages using
-[pouchdb-quick-search](https://github.com/nolanlawson/pouchdb-quick-search).
-This can be improved in many ways, because we are searching through a person's
-memory, not just some arbitrary document collection. For example, we can use
-created assocations and browsing paths to better understand what one is looking
-for.
-
-#### [`src/dev/`](src/dev/): development tools
-
-Tools to help during development. They are not used in production builds.
-
-#### [`src/util/`](src/util): utilities
-
-Contains small generic things, stuff that is not project-specific. Things that
-could perhaps be packaged and published as an NPM module some day.
-
-#### `...`: other stuff
-
-The build process is based on `yarn`, that runs some `npm` commands specified in
-`package.json`, which in turn start the corresponding tasks in
-`gulpfile.babel.js` (transpiled by settings in `.babelrc`).
-
-So much for the code tour. :zzz:  Any questions? :point_up:
+### <a id="worldbrain" href="#worldbrain">WorldBrain</a>
+
+Worldbrain is a browser extension that lets you create and search through your whole web of personal browser history.
+ 
+ - **Store and Index** all the content of every web page you visit. 
+ - **Full-Text-Search** all pages and bookmarks you have visited.
+ - **Full Data Privacy** everything stays on your own computer and will **never ever ever EVER** be shared or used without your consent. Here at worldbrain we are highly concerned about your privacy and security please see our [privacy policy](**TODO**) for more details. 
+
+Lookout for these exiting new features in the future:
+ - **Rate** webpages you visit, add notes, comments and links to similar valuable resources
+ - **Weight** intelligent algorithims to assess the value of a webpage based on your interactions with it. 
+ - **Integrate** with all your favorite services such as facebook, evernote and twitter 
+ - **Share** recommendations, comments, links & ratings with your network.
+ - **Discover** find new content that has high value within your network

```

Comment:
maybe remove "find" as the "**Discover**" acts as the verb like the other sentences here